README.md

```diff
@@ -41,6 +41,29 @@ Here, `<MSGPACK_INCLUDE_PATH>` and `<MMTF_INCLUDE_PATH>` are the paths to the
 
 For your more complicated projects, a `CMakeLists.txt` is included for you.
 
+## Installation
+This is a header only project and hence there is no compilation necessary. If you want to use it in your project you can
+download this repo and add `-I/full/path/to/mmtf-cpp/include` to your `g++` or `clang++` command.  
+*REMINDER* This project depends on [msgpack-c](https://github.com/msgpack/msgpack-c), you must also install that to your system
+or add the `msgpack-c` include directory to your g++ command as well.  
```

Comment:
Following up on @speleo3 comment: let's remove this paragraph. It is indeed just repeating what is already written a few lines above in "How to use MMTF".
A pity that I cannot use the new change suggestion system to suggest a change covering multiple lines... ;-)